devtools/conda-recipe/meta.yaml

```diff
@@ -30,6 +30,7 @@ requirements:
     - protobuf
     - mdtraj
     - setuptools
+    - openbabel
```

Comment:
Do we need to pin the `openbabel` version in the `conda-forge` build as well?

Reply:
We should! I missed that, fixed!